RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -93,7 +109,24 @@
         ttrhBuilderLabel = cms.string('PixelTTRHBuilderWithoutAngle')
     )
 )
-
+eras.trackingPhase2PU140.toModify(detachedQuadStepSeeds,
+    RegionFactoryPSet = dict(
+        RegionPSet = _RegionPsetFomBeamSpotBlock.RegionPSet.clone(
+            ptMin = 0.45,
+            originRadius = 0.7,
+            nSigmaZ = 4.0
+            )
+    ),
+    ClusterCheckPSet = dict(doClusterCheck = False),
+    OrderedHitsFactoryPSet = dict( GeneratorPSet = dict(maxElement = 0 ) ),
+    SeedCreatorPSet = dict( magneticField = '', propagator = 'PropagatorWithMaterial'),
+    SeedMergerPSet = cms.PSet(
+        layerList = cms.PSet(refToPSet_ = cms.string("PixelSeedMergerQuadruplets")),
+	addRemainingTriplets = cms.bool(False),
+	mergeTriplets = cms.bool(True),
+	ttrhBuilderLabel = cms.string('PixelTTRHBuilderWithoutAngle')
```

Comment:
(very minor) add one space to the indentation of `addRemainingTriplets`, `mergeTriplets`, and `ttrhBuilderLabel`.